org.eclipse.lsp4e/src/org/eclipse/lsp4e/IMarkerAttributeComputer.java

```diff
@@ -41,4 +42,14 @@ public interface IMarkerAttributeComputer {
 	 */
 	public void addMarkerAttributesForDiagnostic(Diagnostic diagnostic, @Nullable IDocument document,
 			IResource resource, Map<String, Object> attributes);
+
+	/**
+	 * Computes a string to be used as Marker message.
+	 */
+	public default String computeMarkerMessage(Diagnostic diagnostic) {
```

Comment:
Thanks for the follow-up PR.

Should the default not be the empty String or still need enablement with some configuration property?

So far we only know about one report where this feature is desired, and we know nothing about the rest of the user base.

Reply:
@mickaelistria WDYT? I am leaning towards making/keeping this the default and let providers of language server with extraordinary long diagnostic codes overwrite the default if required.